src/main.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const { createServer } = require('./createServer');
+const { createServer } = require('/createServer').default;
```

Comment:
The import path for `createServer` seems incorrect. It should be `require('./createServer')` instead of `require('/createServer')`. Ensure the path is relative to the current file location.